javascript/swiper.js

```diff
@@ -35,4 +35,16 @@ const portfolioSwiper = new Swiper(".portfolio__swiper", {
       direction: 'horizontal',
     },
   },
+});
+
+const testimonialSwiper = new Swiper(".testimonial__swiper", {
+  slidesPerView: 1,
+  navigation: {
+    nextEl: ".swiper-button-next.testimonial-button",
+    prevEl: ".swiper-button-prev.testimonial-button",
+  },
+  pagination: {
+    el: ".testimonial__boxes__pagination",
+    clickable: true,
+  },
```

Comment:
Sometimes when I changes the current slide from the dot navigation and then click on the arrows one of the arrows becomes disabled, even tho I should be able to change the slide.
![image](https://user-images.githubusercontent.com/68795596/207307904-fe87677c-9458-4c36-8fba-b6023851294a.png)